bundles/org.openhab.binding.sleepiq/src/3rdparty/java/org/openhab/binding/sleepiq/api/CommunicationException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Mark Hilbush
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openhab.binding.sleepiq.api;
```

Comment:
Either the namespace is incorrect, or the file should not have been placed in the 3rd party directory. See https://github.com/openhab/openhab-addons/pull/14165#pullrequestreview-1238535258

Reply:
This should be resolved as a consequence of moving the 3rd party code into the official binding code.